vysledek/style.css

```diff
@@ -0,0 +1,327 @@
+*, ::after, ::before { box-sizing: border-box; }
+
+body {
+    margin: 0;
+    padding: 0;
+    background-color: #f5f5f5;
+    font-family: 'Merriweather', serif;
+    font-size: 16px;
+    line-height: 1.6;
+}
+
+.leading {
+    display: flex;
```

Comment:
Tady bychom si u mobile-first layoutu vystačili i s výchozím `display:block;`, který nám taky řadí položky pod sebe.
Tobě tu flexbox trochu vypomohl, takže nemusíš řešit stylování obrázků v textu 🙂